### PR TITLE
Fix max memory bytes/words mismatch

### PR DESCRIPTION
**Pull request by ddeboer**
While memory_max was defined in MB, the ets table size check was done for number of words instead. This caused the depcache size limit to allow 4 (32 bit) or 8 (64 bit) as much memory use as defined.

### CODE REVIEW

File: src/depcache.erl [hunk 1]
```diff
@@ -353,7 +353,9 @@ get_now() ->
 %% @spec init(Config) -> {ok, State}
 %% @doc Initialize the depcache.  Creates ets tables for the deps, meta and data.  Spawns garbage collector.
 init(Config) ->
-    MemoryMax = 1024 * 1024 * case proplists:get_value(memory_max, Config) of undefined -> ?MEMORY_MAX; Mbs -> Mbs end,
+    MemoryMaxMbs = proplists:get_value(memory_max, Config, ?MEMORY_MAX),
+    MemoryMaxWords = 1024 * 1024 * MemoryMaxMbs / erlang:system_info(wordsize),
```
Comment (mworrell) [inline src/depcache.erl]: Should that be `div` ?  Now the result is a float.

**Review by mworrell — CHANGES_REQUESTED** [review]

**Comment by ddeboer** [comment]
Changed division from `/` to `div`.

**Review by mworrell — APPROVED** [review]

**Comment by arjan** [comment]
I can add you as package owners on hex.pm for depcache, which email addresses do I need for that?

**Comment by ddeboer** [comment]
See https://github.com/ddeboer 😉 

I don’t think Zotonic installs depcache through Hex. Or are you talking about other users?

**Pull request merged**